README.md

```diff
@@ -129,6 +129,19 @@ the steps accordingly.
 
 That's it!  The service should now be running and working.
 
+## Upgrade
+
+Upgrade the service to the latest git version with these stepos:
+
+```
+cd /opt/ff-node-monitor/src/
+sudo systemctl stop ff-node-monitor
```

Comment:
Actually this is not necessary. If you replace this by `sudo rm /opt/ff-node-monitor/src/target/release/ff-node-monitor`, you can change the `start` below to a `restart` and reduce the downtime.